include/OpenSpaceToolkit/Astrodynamics/Trajectory/Sequence.hpp

```diff
@@ -35,6 +35,7 @@ class Sequence
     struct Solution
     {
         Array<Segment::Solution> segmentSolutions;
+        bool sequenceIsComplete;
```

Comment:
nit: what do you think about something like `sequenceExecutionIsComplete`. It took me a second to understand that this didn't mean something like "the sequence is fully constructed and valid for execution"